backend/command/src/shared/validators/is-bigger-than.ts

```diff
@@ -0,0 +1,33 @@
+// Full credits to:
+// https://stackoverflow.com/questions/59980341/can-i-compare-number-variables-with-class-validator/67815042#67815042
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+} from 'class-validator';
+
+export function IsBiggerThan(
```

Comment:
Test(s)

Reply:
Good point! Is in TODO.

Reply:
Added tests!

![image](https://user-images.githubusercontent.com/47084064/217565056-4598bf51-1256-4a01-a22f-6ce115f18d9d.png)

Close conversation if it's fine :)